packages/integration-tests/src/tests/console/bootstrap.test.ts

```diff
@@ -82,6 +83,18 @@ describe('smoke testing for console admin account creation and sign-in', () => {
     expect(page.url()).toBe(new URL('console/get-started', logtoConsoleUrl).href);
   });
 
+  it('should have html attributes "lang=en" and "dir=ltr" by default', async () => {
```

Comment:
do we have similar tests in experience?

Reply:
Good point. I don't think I did. Will merge this one first, as team has been complaining about it. Will add more test cases in a follow-up PR.